src/lib.rs

```diff
@@ -943,7 +958,7 @@ fn fill_todo(
                         children
                             .retain(|x| !x.file_name().unwrap().to_str().unwrap().starts_with("."));
```

Comment:
Nit: we could use newly added `file_name` member here, though I don't have a way to test the performance impact of that on real-world code.

Reply:
Same goes for this part: https://github.com/rust-lang/glob/blob/76cafd1527ba55e1f9a1b9002dda27fe10eff53d/src/lib.rs#L473
which seems a bit more relevant to the "hot path".